src/components/bs5/footer/footer.data.json

```diff
@@ -32,15 +33,14 @@
     },
     "acknowledgements": [
         {
-            "title": "",
-            "content": "<p>We pay our respects to the Aboriginal and Torres Strait Islander ancestors of this land, their spirits and their legacy. The foundations laid by these ancestors—our First Nations peoples—give strength, inspiration and courage to current and future generations towards creating a better Queensland.</p>"
+            "title": "Acknowledgement of Country",
+            "content": "<p>Queensland Government acknowledges the Traditional Owners of the land and pays respect to Elders past, present and future.</p>"
         }
     ],
     "copyright": {
         "hasBorderColumnDisabled": false,
         "content": "&copy; The State of Queensland",
-        "yearFrom": "1995",
-        "currentYear": "2024"
```

Comment:
add current year back to the copyright text

Reply:
Great point, Katja.

The reason I removed currentYear from json data is because the current calendar year should be dynamically generated from a script, without any data input from human or CMS (Squiz Matrix).

On this PR, I have also updated the template files (footer.hbs and footerForgov.hbs) to use `{{formatDateOrToday "" "YYYY" }} `instead of the json data `copyright.currentYear`.